yarn-project/simulator/src/public/public_processor.test.ts

```diff
@@ -282,683 +238,6 @@ describe('public_processor', () => {
       expect(handler.addNewTx).toHaveBeenCalledTimes(0);
     });
 
-    it('rolls back app logic db updates on failed public execution, but persists setup', async function () {
```

Comment:
Was this too difficult to recreate?

Reply:
It was recreated! Maybe I just didn't put them in the same order ...

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/9565/files#diff-f01956ef9dcfeec27e994abb796992b50bcf02429913a2f9469dfb8fdec839d4R284